h/tweens.py

```diff
@@ -75,7 +76,7 @@ def redirect_tween_factory(handler, registry, redirects=None):
         # N.B. If we fail to load or parse the redirects file, the application
         # will fail to boot. This is deliberate: a missing/corrupt redirects
         # file should result in a healthcheck failure.
-        with open("h/redirects", encoding="utf-8") as fp:
+        with open(resource_filename("h", "redirects"), encoding="utf-8") as fp:
```

Comment:
This allows me to run the functests without it being super sensitive to the working directory